tests/everest/test_everserver.py

```diff
@@ -271,3 +274,52 @@ def test_everserver_status_max_batch_num(
         filter_out_gradient=False, batches=None
     )
     assert {data.batch for data in snapshot.simulation_data} == {0}
+
+
+@patch("sys.argv", ["name", "--config-file", "config_minimal.yml"])
+@patch("everest.detached.jobs.everserver._configure_loggers")
+@patch("everest.detached.jobs.everserver._generate_authentication")
+@patch(
+    "everest.detached.jobs.everserver._generate_certificate",
+    return_value=(None, None, None),
+)
+@patch(
+    "everest.detached.jobs.everserver._find_open_port",
+    return_value=42,
+)
+@patch("everest.detached.jobs.everserver._write_hostfile")
+@patch("everest.detached.jobs.everserver._everserver_thread")
+def test_everserver_status_contains_max_runtime_failure(
+    _1, _2, _3, _4, _5, _6, change_to_tmpdir, min_config
+):
+    config_file = "config_minimal.yml"
+
+    # Add sleep job
+    Path("sleep.py").write_text(
+        "#!/usr/bin/env python\nimport time\ntime.sleep(5)", encoding="utf-8"
+    )
+    st = os.stat("sleep.py")
+    os.chmod("sleep.py", st.st_mode | stat.S_IEXEC)
+    Path("SLEEP").write_text(
+        "EXECUTABLE  sleep.py\n\nMIN_ARG 0\nMAX_ARG 6", encoding="utf-8"
+    )
+
+    # 2 second max runtime
+    min_config["simulator"] = {"max_runtime": 2}
+    min_config["forward_model"] = ["sleep"]
+    min_config["install_jobs"] = [{"name": "sleep", "source": "SLEEP"}]
```

Comment:
This can be simplified to:

```suggestion
        # Add sleep job
    Path("SLEEP_job").write_text("EXECUTABLE sleep", encoding="utf-8")

    # 2 second max runtime
    min_config["simulator"] = {"max_runtime": 2}
    min_config["forward_model"] = ["sleep 5"]
    min_config["install_jobs"] = [{"name": "sleep", "source": "SLEEP_job"}]
```

Reply:
:exploding_head: I did not know `sleep` was a UNIX command, this will change my file forever!